components/build-service/overlays/kcp-unstable/apiexport.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: application-service
+spec:
+  permissionClaims:
+    - group: ""
+      resource: "secrets"
+    - group: ""
+      resource: "configmaps"
+    - group: ""
+      resource: "namespaces"
+    - group: ""
+      resource: "events"
+    - group: ""
+      resource: "serviceaccounts"
+    - group: "tekton.dev"
+      resource: "pipelineruns"
+      identityHash: pipeline-service
```

Comment:
shouldn't this contain the identity hash as well?

Reply:
Currently there is no deployment in kcp-unstable, but yeah comment would be great